birdhouse/config/thredds/default.env

```diff
@@ -6,10 +6,28 @@ export THREDDS_DOCKER=pavics/thredds-docker
 export THREDDS_IMAGE="${THREDDS_DOCKER}:${THREDDS_VERSION}"
 export THREDDS_ORGANIZATION="Birdhouse"
 
+# All default values for data locations are set to ensure backward compatibility
+export DATA_LOCATION_FOR_THREDDS_ON_CONTAINER='/pavics-ncml'
+export RAW_DATA_LOCATION_FOR_THREDDS_ON_CONTAINER='/pavics-data'
+export DATA_LOCATION_FOR_THREDDS_ON_HOST='${DATA_PERSIST_ROOT}/ncml'  # this default is for backward compatibility 
+export RAW_DATA_LOCATION_FOR_THREDDS_ON_HOST='${DATA_PERSIST_ROOT}/datasets'  # this default is for backward compatibility
+export DATA_LOCATION_NAME='Datasets'  # this default is for backward compatibility 
+export RAW_DATA_LOCATION_NAME='Birdhouse'  # this default is for backward compatibility 
+export DATA_LOCATION_NAME_lower=$(echo "$DATA_LOCATION_NAME" | tr '[:upper:]' '[:lower:]')
```

Comment:
This should be single quoted so that it's not evaluated too soon (since it's one of the DELAYED_EVAL variables now)

And the line below as well